content/sdk/10.js/00.ethers/00.index.md

```diff
@@ -68,7 +93,7 @@ to: https://join.zksync.dev
 ---
 title: Support Desk
 icon: i-heroicons-computer-desktop
-to: https://github.com/zksync-sdk/sdk-docs/blob/main/SUPPORT.md
+to: https://zksync.io/contact
```

Comment:
Let's keep this pointing to the SUPPORT.md file. This file we have direct editing capabilities for. This also follows the GitHub open source community guidelines for project files. We don't have any access to edit that zksync.io/contact page.

Reply:
I have removed the entire page from this section